src/js/dataMapping/agencyV2/visualizations/totalObligationsOverTime.js

```diff
@@ -13,9 +13,25 @@ export const xLabelHeightPlusPadding = xLabelHeight + xLabelSpacing;
 
 export const yOffsetForPathStrokeWidth = (pathStrokeWidth + 1) / 2;
 
+export const defaultHeight = 179;
+
 export const defaultPadding = {
     top: 24,
     bottom: 24,
     right: 30,
     left: 30
 };
+
+// $color-secondary
+export const pathStopColorRed = '#e31c3d';
+// $color-cool-blue
+export const pathStopColorBlue = '#205493';
+// $color-secondary-lightest
+export const areaPathStopColorRed = '#f9dede';
+// $color-cool-blue-lightest
+export const areaPathStopColorBlue = '#dce4ef';
```

Comment:
Can we not use the global color variables here that are commented instead of the hex values?

Reply:
We could migrate these to class names, then update the code to apply different class names based on the different scenarios.

But no, we cannot use the variables in the JS files.